### PR TITLE
Graph generation test case for node to set to node

### PR DESCRIPTION
**Pull request by dvargas92495**
`graph = A >> {B, C} >> D`

### CODE REVIEW

File: ee/codegen/src/generators/graph-attribute.ts [hunk 1]
```diff
@@ -316,14 +318,22 @@ export class GraphAttribute extends AstNode {
...
-                type: "right_shift",
-                lhs: mutableAst,
-                rhs: { type: "node_reference", reference: targetNode },
-              };
-            }
+            return;
+          }
+
+          const lhsTerminals = getAstTerminals(mutableAst.lhs);
+          const lhsTerminal = lhsTerminals[0];
```
Comment (dvargasfuertes) [inline ee/codegen/src/generators/graph-attribute.ts:325]: Even the sdk itself doesn't support `{A, B} >> {C, D}` yet. We'll need to make sdk-side changes first to do so

**Review by NgoHarrison — APPROVED** [review]
✅

File: ee/codegen/src/generators/graph-attribute.ts (hunk 1 above)
Comment (noanflaherty) [reply ee/codegen/src/generators/graph-attribute.ts:325]: Is it easy to detect this case and throw an error if we do? My goal is to have code that self-documents this edge case and makes it easy for a future developer to know where to update if/when we do support this.

**Review by noanflaherty — APPROVED** [review]

**Review by kirawoods — COMMENTED** [review]
✅

**Pull request merged**